installer/mac/intro.html

```diff
@@ -1,11 +1,18 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>Welcome to the Symphony Installer</title>
+	<title>Thanks for downloading Symphony</title>
 </head>
 <body>
 <br/>
 <br/>
-<p style="font-size: 13px; font-family: Sans-Serif; color: #000">&nbsp;&nbsp;&nbsp;We'll walk you through the steps to install Symphony</p>
+<p style="font-size: 13px; font-family: Sans-Serif; color: #000">	
```

Comment:
I'd like for this to use the same fonts/fallbacks that the web app is using. Can you please take a look?

Reply:
Installer doesn't support custom fonts. Only the ones available by default on the system.

Reply:
Yes. Please refer to https://css-tricks.com/snippets/css/system-font-stack/